Dockerfile.template

```diff
@@ -1,16 +1,18 @@
+# syntax=docker/dockerfile:1.2
+
 ARG OS_VERSION
 ARG DEVICE_TYPE=%%BALENA_MACHINE_NAME%%
 
 FROM resin/resinos:${OS_VERSION}.dev-${DEVICE_TYPE}
 
+COPY ./entry.sh /entry.sh
+COPY ./conf/systemd-watchdog.conf /etc/systemd/system.conf.d/watchdog.conf
+
 ENV container=docker
 
 VOLUME /mnt/boot \
        /mnt/state \
        /mnt/data
 
-COPY ./conf/systemd-watchdog.conf /etc/systemd/system.conf.d/watchdog.conf
-COPY ./aufs2overlay.sh /aufs2overlay
```

Comment:
If we remove the legacy scripts let's also remove the aufs2overlay one as it is no longer used when using the compose file.